packages/orchestrator/internal/sandbox/signal.go

```diff
@@ -0,0 +1,23 @@
+package sandbox
+
+import (
+	"context"
+	"os"
+	"os/signal"
+)
+
+func listenProcessSignals(
```

Comment:
is this useful, I guess there aren't almost any signals. But I think if we did something similar inside the sandbox it would be really cool

Reply:
Btw this will include all signals right? Not only to the one specific sandbox

Reply:
Exactly, im thinking i should push this further and, per a discussion with Vasek, listen to signals for specific processes in the sandbox.

Reply:
> Btw this will include all signals right? Not only to the one specific sandbox

All signals of the FC PID yeah

Reply:
Maybe I miss something, but i didn't any filter on FC PID.